ts/components/ConversationListItem.tsx

```diff
@@ -95,7 +137,11 @@ const HeaderItem = (props: {
           profileName={profileName}
         />
       </div>
-      {pinIcon}
+
+      <StyledConversationListItemIconWrapper>
+        {pinIcon}
+        <NotificationSettingIcon />
```

Comment:
you need to define a key for those two items
.you should be able to start the app with yarn start only to see this error